service-front/app/test/CommonTest/Form/AbstractFormTest.php

```diff
@@ -20,7 +23,7 @@ protected function setUp(): void
     {
         $guardProphecy = $this->prophesize(CsrfGuardInterface::class);
 
-        $this->form = new class('testConcreteForm', $guardProphecy->reveal()) extends AbstractForm {
+        $this->form = new class ('testConcreteForm', $guardProphecy->reveal()) extends AbstractForm {
```

Comment:
```suggestion
        $this->form = new class('testConcreteForm', $guardProphecy->reveal()) extends AbstractForm {
```
Did your IDE suggest this?

Reply:
Psalm did I think

Reply:
Will double check later

Reply:
If it's been suggested by a linter then I guess it's what we should be doing.

Reply:
![Screenshot 2022-09-14 at 14 40 33](https://user-images.githubusercontent.com/84718230/190170065-dc94f6cb-e766-4cef-a173-ab25e2a5c0d9.png)
It's in my IDE but also was raised by phpcs in the github action D:

Reply:
spotted. it. it is a PSR12 thing. go figure.